coverage/coverage-final.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This folder is not supposed to be pushed, you might want to list it in your `.gitignore`